templates/tensorflow/r1.15/base.libsonnet

```diff
@@ -0,0 +1,14 @@
+local base = import "../base.libsonnet";
+local mixins = import "../../mixins.libsonnet";
+
+{
+  LegacyTpuTest:: base.LegacyTpuTest {
+    frameworkPrefix: "tf-r1.15",
+    tpuVersion: "1.15",
+    imageTag: "1.15",
+  },
+  Convergence:: mixins.Convergence {
+    # Run at 1:00 PST on Saturday
+    schedule: "0 8 * * 6"
```

Comment:
Do we have to run these tests this frequently? Do we have to automate them at all, or can we just run them manually whenever we patch the 1.15 image?

I'm just asking -- I don't have a clear idea of what we need myself.

Reply:
Hmm, for now I don't think so. When 1.15.3 starts to roll out we might need to ramp it up - we'll probably have moved most BCT tests by then right?